scripts/docker-entrypoint.sh

```diff
@@ -35,16 +35,34 @@ mkdir -p $tls_cert_root
 tomcat_cert_root="${pega_root}/tomcatcerts"
 mkdir -p $tomcat_cert_root
 
-prlog4j2="${config_root}/prlog4j2.xml"
-prconfig="${config_root}/prconfig.xml"
-context_xml="${config_root}/context.xml"
-server_xml="${config_root}/server.xml"
-web_xml="${config_root}/web.xml"
-tomcatusers_xml="${config_root}/tomcat-users.xml"
-catalina_properties="${config_root}/catalina.properties"
-prbootstrap_properties="${config_root}/prbootstrap.properties"
-java_security_overwrite="${config_root}/java.security.overwrite"
-tomcat_web_xml="${config_root}/tomcat-web.xml"
+decompressed_root="${pega_root}/decompressedconfig"
+mkdir -p $decompressed_root
```

Comment:
I think this folder is created in the Dockerfile. Is this mkdir going to be different?